components/editor/vars.ftd

```diff
@@ -204,7 +204,7 @@ ide_dispatch_event(name, data)
 
 
 ;; full-name of current file
--- optional string $current-file:
+-- optional string $current-file: sfd
```

Comment:
If this is intentional, explain in a comment why sfd or why it can't be left null.